UI/properties-view.cpp

```diff
@@ -208,6 +208,26 @@ void OBSPropertiesView::resizeEvent(QResizeEvent *event)
 	VScrollArea::resizeEvent(event);
 }
 
+bool OBSPropertiesView::eventFilter(QObject * obj, QEvent * event)
+{
+	if (event->type() == QEvent::Wheel) {
+		if (obj->inherits("QSlider")) {
```

Comment:
Small nitpick - these three conditionals could technically all be one with an `or` comparison (`||`) to cut down on duplicate code for the three types of properties.

You could even go one level further and merge it into the event type comparison.